src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/sources/sass/libs/_vars.scss

```diff
@@ -1,6 +1,6 @@
 // Misc.
 	$misc: (
-		z-index-base:		10000
+		z-index-base:		1000
```

Comment:
@rart this variable is used only in the sidebar, so no other element z-index will be affected